src/Playroom/SettingsPanel/SettingsPanel.tsx

```diff
@@ -71,12 +71,14 @@ const KeyboardShortcut = ({
   ));
 
   return (
-    <div className={styles.keyboardShortcutRow}>
-      <Text>{description}</Text>
+    <Inline space="small" alignY="center">
+      <div className={styles.keyboardShortcutTitle}>
+        <Text>{description}</Text>
+      </div>
       <Text size={isMac() ? 'large' : 'standard'}>
-        <Inline space="xxsmall">{shortcutSegments}</Inline>
+        <div className={styles.keyboardShortcutKeys}>{shortcutSegments}</div>
       </Text>
-    </div>
+    </Inline>
```

Comment:
My bad if we already discussed this, but why use grid here instead of sticking with `Inline`?